### PR TITLE
Add custom font example

### PR DESCRIPTION
**Pull request by bradgarropy**
image

### CODE REVIEW

File: examples/react-custom-fonts/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+const cloudinary = new Cloudinary({
+    cloud: {
+        cloudName: import.meta.env.VITE_CLOUDINARY_CLOUD_NAME,
+    },
+})
+
+const cldImage = cloudinary
+    .image("examples/sneakers_cuuqb1.jpg")
```
Comment (colbyfayock) [inline examples/react-custom-fonts/src/App.jsx:19]: can you update this public ID to `cld-sample-5`? im going to start encouraging use of images from the `samples` directory or samples that exist in all new accounts ot make it easier for people to spin these projects up

**Comment by colbyfayock** [comment]
hey @bradgarropy can you add a README similar to the other projects that just instructs how to set up the project?

**Comment by bradgarropy** [comment]
Added a `README` and a `.env.example` file!

**Comment by colbyfayock** [comment]
thanks @bradgarropy

**Pull request merged**